game/src/core/Component.java

```diff
@@ -1,48 +1,17 @@
 package core;
 
-import java.util.logging.Logger;
-
 /**
  * Components store the data (or attributes) for an associated {@link Entity}.
  *
- * <p>This class is the abstract base class for each component.
- *
- * <p>Each component is linked to exactly one entity. Use {@link #entity} to get the associated
- * entity of the component.
+ * <p>This interface needs to be implemented by each component.
  *
- * <p>Each component will automatically add itself to the associated entity using {@link
- * Entity#addComponent}.
+ * <p>Each component can be linked to exactly zero, one or multiple entities. Use {@link
```

Comment:
```suggestion
 * <p>Each component can be linked to zero to n entities. Use {@link
```

Muss nicht, finde ich aber schöner :wink: